packages/components/src/components/Switch/index.tsx

```diff
@@ -31,6 +31,7 @@ const SwitchInput = styled.input(
     width: 0,
     opacity: 0,
     position: 'absolute',
+    left: -100,
```

Comment:
workbench-theme has styles that bring the input back :P, so this just takes it out of screen